components/calendar/src/astronomy.rs

```diff
@@ -978,10 +977,15 @@ impl Astronomical {
 
     #[allow(dead_code, clippy::unwrap_used, clippy::eq_op)]
     pub(crate) fn moonlag(date: Moment, location: Location) -> Option<f64> {
```

Comment:
nit: please document why this would return None (and also what this does in general)

Reply:
fixed and added reference